.github/workflows/pipeline-main.yml

```diff
@@ -12,6 +12,18 @@ jobs:
     uses: ./.github/workflows/pipeline-core.yml
   other:
     uses: ./.github/workflows/pipeline-other.yml
+  docs:
```

Comment:
If this went in `release.yml`, it could benefit from `alert_failure`, but up to you. (it could `needs: [release]`)

Reply:
Good idea. I've moved this over there.